lib/employee-benefit-plan.ts

```diff
@@ -3,8 +3,11 @@ import { Record } from "./flatfile";
 import { prismaClient } from "./prisma-client";
 import { DateTime } from "luxon";
 import { upsertBenefitPlan } from "./benefit-plan";
+import { RecordsWithLinks } from "@flatfile/api/api";
 
-export const validEmployeeBenefitPlanRecords = async (records: Record[]) => {
```

Comment:
It's time to cut this function, the records we get back should be valid

Reply:
removed